tests/travis-common-script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
```

Comment:
I like this change (this is a bugfix), but this also breaks the current functionality. The current code depends on that the script does not exit early even in case of error: for example `S=$?`.

This should be resolved somehow.

Reply:
I think this is not necessary a bugfix. 
I can change my opinion, but for this I need to understand why we did not want to return early if there is an error (return with not 0), imho this can be a real bug.